schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts SMALLINT NOT NULL,
     neutered BOOLEAN NOT NULL,
     weight_kg FLOAT NOT NULL,
+    species VARCHAR(20) NOT NULL DEFAULT '',
```

Comment:
- [ ] <b>REQUIRED:</b> The project require you to add column `species` to table animals by `ALTER TABLE ` not by changing the previous query. Kindly do that.

Reply:
no, it does not, and I quote:

> Add a column species of type string to your animals table. Modify your schema.sql file.